bin/brew

```diff
@@ -1,7 +1,10 @@
-#!/bin/sh
+#!/bin/bash
 
 chdir() {
-  cd "$@" >/dev/null
+  cd "$@" >/dev/null || {
+    echo "Error: failed to cd to " "$@" "!" >&2
```

Comment:
I think we have enough instances of “print something to `stderr` and exit with failure” to warrant a shell function, e.g.,

```shell
odie() {
  printf "%s\n" "$*" >&2
  exit 1
}
```

and then use it here like:

```shell
cd "$@" >/dev/null || odie "Error: failed to cd to $*!"
```